app/src/routes/adminRoute.jsx

```diff
@@ -12,13 +16,28 @@ export const adminRoute = (Component) =>
       authorized: isAuthorizedSelector(state),
       accountRole: userAccountRoleSelector(state),
       projectId: activeProjectSelector(state),
+      isAdminAccess: isAdminAccessSelector(state),
     }),
     (dispatch) => ({
       redirect: (projectId) =>
         dispatch(rfrRedirect({ type: PROJECT_DASHBOARD_PAGE, payload: { projectId } })),
+      setAdminAccess: () => dispatch(setAdminAccessAction()),
```

Comment:
setAdminAccess: setAdminAccessAction

Reply:
The same as https://github.com/reportportal/service-ui/pull/1246#discussion_r265908607